HELM-341: Documentation formatting cleanup (#484)
OpenNMS/opennms-helm: docs/modules/panel_configuration/pages/dynamic-dashboard.adoc

* xref:pc-filter-panel[filter panels]

[[pc-template-filters]]

> == Using variables to create dynamic dashboards

Reviewer: ```suggestion
== Use variables to create dynamic dashboards
```

.. A label must be unique to display data in the panel.
+
image::pc-node-Final.png[Node final, 300]

. Save your dashboard.
. Click the *Node* drop-down you created to select the nodes whose data you want to display:
+
image::pc-select-nodes.png[Node Select, 500]

[[pc-filter-panel]]
== Create a filter panel

A filter panel lets you configure a series of variables to put on a dashboard.
While xref:pc-template-filters[template variables] work with the performance datasource, the filter panel coordinates with any panels in the dashboard that use an xref:datasources:entity_datasource.adoc[entities datasource] to automatically apply filters that match those variables.

Before creating a filter panel, make sure you have a panel on the dashboard that uses an entities datasource (see xref:getting_started:basic_walkthrough.adoc#bw-panel-create[create a panel]).

This procedure creates a panel that lets you filter on alarm type and alarms that are acknowledged or in a situation.

To create a filter panel, follow these steps:

. In the dashboard, click the *Add Panel* icon.
. Click *Choose Visualization* and choose *Filter Panel*.

> . In the *Filter Columns* area, choose an entities datasource.

Reviewer 2: ```suggestion
. In the Filter Columns area, choose an entities datasource.
```

+
image::pc-node-Final.png[Node final, 300]

. Save your dashboard.
. Click the *Node* drop-down you created to select the nodes whose data you want to display:
+
image::pc-select-nodes.png[Node Select, 500]

[[pc-filter-panel]]

> == Creating a filter panel

Reviewer: ```suggestion
== Create a filter panel
```

A filter panel lets you configure a series of variables to put on a dashboard.
While xref:pc-template-filters[template variables] work with the performance datasource, the filter panel coordinates with any panels in the dashboard that use an xref:datasources:entity_datasource.adoc[entities datasource], to automatically apply filters that match those variables.

Before creating a filter panel, make sure you have a panel on the dashboard that uses an entities datasource (see xref:getting_started:basic_walkthrough.adoc#bw-panel-create[create a panel]).

> This procedure creates a panel that will let you filter on alarm type and alarms that are acknowledged or in a situation.

Reviewer: ```suggestion
This procedure creates a panel that lets you filter on alarm type and alarms that are acknowledged or in a situation.
```

+
image::pc-filter-panel-cust.png[Filter Panel, 300]

> .  Repeat steps 4–7 to add additional filters.

Reviewer 2: ```suggestion
. Repeat steps 4–7 to add additional filters.
```

+
image::pc-filter-panel-cust.png[Filter Panel, 300]

> . Add additional filters by repeating steps 4–7.

Reviewer: ```suggestion
.  Repeat steps 4–7 to add additional filters.
```
